docs/gettinginvolved/benchmarking.rst

```diff
@@ -0,0 +1,64 @@
+.. _benchmarking:
+
+Benchmarking
+============
+
+Flocker includes a tool for benchmarking operations.
+It needs the appropriate ssh-key added to a running ssh-agent.
+It is called like this:
+
+.. prompt:: bash $
+
+   python benchmark/benchmark_control.py <options>
```

Comment:
Nah, I'm just lazy. :laughing: 

Rather than an entrypoint, I think the code should follow the model used in `admin` directory, so the command would invoked as `benchmark/benchmark_control <options>`.

However, I might put that question out for discussion. Should benchmarking be available on deployed clusters?